nss-day-cohort-22/Class-Website
# Kn adding photos to json

## Description of Proposed Changes

- Added student images to the JSON file 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout kn-AddingPhotosToJSON

```
check that photos appear for all students and that the fun ones appear when you click "Learn More"
Courtney is the only one who didn't have images, I reached out to her about what photos we should use for her.

I added a box shadow behind the fun photo, but I'm not 100% sure that I like it. I felt it needed something to make the photo stand out a little. Open to any suggestions

## Impacted Areas in Application

List general components of the application that this PR will affect:
* Student Images


## Mentions @username

Tag users that need to review this code
